react/util/updateMediatorBlocks.js

```diff
@@ -0,0 +1,11 @@
+import mediator from 'lib/mediator.coffee';
+import Blocks from 'collections/blocks.coffee';
+
+export default ({ blocks }) => {
+  mediator.shared.blocks = new Blocks(blocks.map(k => ({
+    id: k.id,
+    class: k.__typename,
+    base_class: 'Block',
+    unloaded: true,
+  })));
+};
```

Comment:
Just wondering why this is needed?

Reply:
I can remove it, but eventually the next page will load and `mediator.blocks` will need to be updated.